src/Kunstmaan/FixturesBundle/composer.json

```diff
@@ -22,6 +22,7 @@
         "doctrine/doctrine-fixtures-bundle": "~2.2",
         "fzaninotto/faker": "^1.6",
         "kunstmaan/admin-bundle": "~5.2",
+        "symfony/mime": "^4.4",
         "kunstmaan/media-bundle": "~5.2",
```

Comment:
I would bump this to ^5.6 to make sure we have the mime_type service changes from the media bundle available.